src/main/java/com/salesforce/dataloader/process/ProcessRunner.java

```diff
@@ -98,6 +98,15 @@
 
     private Controller controller;
 
+    static {
+        try {
+            Controller.initLog();
+        } catch (ControllerInitializationException e) {
+            System.out.println("ProcessRunner: Controller not initialized" + e );
```

Comment:
Same as before, we should pipe it into the log

Reply:
Resorting to System.out.println() because the log is not initialized if an exception is thrown.